src/commands.c

```diff
@@ -86,6 +86,12 @@ static void cmd_start(Tox *m, int friendnum, int argc, char (*argv)[MAX_COMMAND_
     }
 
     tox_friend_send_message(m, friendnum, TOX_MESSAGE_TYPE_NORMAL, (uint8_t *) outmsg, strlen(outmsg), NULL);
+
+    char *play_msg = "CSBot Has started, It's time to PLAY!";
+    for(int i = Tox_Bot.num_friends; i >= 0; i-- ) {
```

Comment:
It doesn't make a difference (right now), but the proper way would be to call to tox_friend_get_connection_status() before sending the message